docs/RECIPES.md

```diff
@@ -155,6 +155,10 @@ You will also need:
 - A Partner Connect API Key
   - An API Key can be created in Red Hat Partner Connect at the following [URL](https://connect.redhat.com/account/api-keys)
   - Required for submit
+- A Docker Auth file, can be specified multiple ways
+  - If PFLT_DOCKERCONFIG envvar is set, or --docker-config is passed on the command line, this will be used to pull images.
```

Comment:
Nothing is changing how we read in the file we send to `pyxis` correct? Do we need to note that for submit this must be set? Or do you think that's already implied by the cmd checking for the flag/env and erroring on `--submit` if not provided? I'm fine either way, just want clarity.

Reply:
Obviously this comment is out of date.

Reply:
Yeah this was from when both env's were listed I will resolve this comment.